pkg/manifests/nginx.go

```diff
@@ -339,6 +334,18 @@ func newNginxIngressControllerService(conf *config.Config, ingressConfig *NginxI
 			},
 		},
 	}
+
+	if !ingressConfig.HTTPDisabled {
+		ret.Spec.Ports = append([]corev1.ServicePort{
```

Comment:
small nit on go style -

rather than have append(new item, originalList...)

convention is to have

append(originalList, new item)

Reply:
this is better because it doesn't change the ordering from what it originally was.